2023-09-29 03:24:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "CREATE TABLE LIKES ( ID INTEGER GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY, FILM_ID INTEGER REFERENCES FILMS(ID), USER_ID INTEGER REFERENCES USERS(ID), )[*]"; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE LIKES ( ID INTEGER GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY, FILM_ID INTEGER REFERENCES FILMS(ID), USER_ID INTEGER REFERENCES USERS(ID), )[*]"; expected "identifier"; SQL statement:
CREATE TABLE likes ( id INTEGER GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY, film_id INTEGER REFERENCES films(id), user_id INTEGER REFERENCES users(id), ) [42001-200]
2023-09-29 03:25:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "USERS" уже существует
Table "USERS" already exists; SQL statement:
CREATE TABLE users ( id INTEGER GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY, email VARCHAR, login VARCHAR, name VARCHAR, birthday TIMESTAMP ) [42101-200]
2023-09-29 03:38:06 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Илья Шейман/dev/filmorate/java-filmorate/db/filmorate.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Илья Шейман/dev/filmorate/java-filmorate/db/filmorate.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476)
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712)
	at java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:264)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Илья Шейман/dev/filmorate/java-filmorate/db/filmorate.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Илья Шейман/dev/filmorate/java-filmorate/db/filmorate.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 34 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/Илья Шейман/dev/filmorate/java-filmorate/db/filmorate.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:166)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 28 more
Caused by: java.nio.channels.OverlappingFileLockException
	at java.base/sun.nio.ch.FileLockTable.checkList(FileLockTable.java:229)
	at java.base/sun.nio.ch.FileLockTable.add(FileLockTable.java:123)
	at java.base/sun.nio.ch.FileChannelImpl.tryLock(FileChannelImpl.java:1154)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at java.base/java.nio.channels.FileChannel.tryLock(FileChannel.java:1165)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	... 29 more
2023-09-29 11:33:29 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "NSERT[*] INTO USERS (EMAIL, LOGIN, NAME, BIRTHDAY) VALUES (""mail@mail.ru"", ""dolore"", ""Nick Name"", ""1946-08-20"")"
Syntax error in SQL statement "NSERT[*] INTO USERS (EMAIL, LOGIN, NAME, BIRTHDAY) VALUES (""mail@mail.ru"", ""dolore"", ""Nick Name"", ""1946-08-20"")"; SQL statement:
NSERT INTO users (email, login, name, birthday) VALUES ("mail@mail.ru", "dolore", "Nick Name", "1946-08-20") [42000-200]
2023-09-29 11:35:40 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "NSERT[*] INTO USERS (EMAIL, LOGIN, NAME, BIRTHDAY) VALUES (""mail@mail.ru"", ""dolore"", ""Nick Name"", 1946-08-20 00:00:00.0)"
Syntax error in SQL statement "NSERT[*] INTO USERS (EMAIL, LOGIN, NAME, BIRTHDAY) VALUES (""mail@mail.ru"", ""dolore"", ""Nick Name"", 1946-08-20 00:00:00.0)"; SQL statement:
NSERT INTO users (email, login, name, birthday) VALUES ("mail@mail.ru", "dolore", "Nick Name", 1946-08-20 00:00:00.0) [42000-200]
2023-09-29 11:35:41 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "NSERT[*] INTO USERS (EMAIL, LOGIN, NAME, BIRTHDAY) VALUES (""mail@mail.ru"", ""dolore"", ""Nick Name"", 1946-08-20 00:00:00.0)"
Syntax error in SQL statement "NSERT[*] INTO USERS (EMAIL, LOGIN, NAME, BIRTHDAY) VALUES (""mail@mail.ru"", ""dolore"", ""Nick Name"", 1946-08-20 00:00:00.0)"; SQL statement:
NSERT INTO users (email, login, name, birthday) VALUES ("mail@mail.ru", "dolore", "Nick Name", 1946-08-20 00:00:00.0) [42000-200]
2023-09-29 11:35:41 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "NSERT[*] INTO USERS (EMAIL, LOGIN, NAME, BIRTHDAY) VALUES (""mail@mail.ru"", ""dolore"", ""Nick Name"", 1946-08-20 00:00:00.0)"
Syntax error in SQL statement "NSERT[*] INTO USERS (EMAIL, LOGIN, NAME, BIRTHDAY) VALUES (""mail@mail.ru"", ""dolore"", ""Nick Name"", 1946-08-20 00:00:00.0)"; SQL statement:
NSERT INTO users (email, login, name, birthday) VALUES ("mail@mail.ru", "dolore", "Nick Name", 1946-08-20 00:00:00.0) [42000-200]
2023-09-30 18:02:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "USERS" уже существует
Table "USERS" already exists; SQL statement:
CREATE TABLE users ( id INTEGER GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY, email VARCHAR(50), login VARCHAR(50), name VARCHAR(50), birthday TIMESTAMP ) [42101-200]
